newrelicutil.go

```diff
@@ -45,43 +45,43 @@ func WrapHandler(app newrelic.Application, name string, handler http.Handler) ht
 }
 
 // Segment returns the New Relic Segment object from context.
-func Segment(ctx context.Context) newrelic.Segment {
-	if sgm, ok := ctx.Value(segment).(newrelic.Segment); ok {
+func Segment(ctx context.Context) *newrelic.Segment {
```

Comment:
Why did you change the signature of the function?

Reply:
because newrelic v2 changed this signature